clicktests/helpers.js

```diff
@@ -105,3 +107,7 @@ helpers.selectAllText = function(page) {
   helpers.log('Trying to select all in focused element (ctrl-A)');
   page.sendEvent('keypress', page.event.key.A, null, null, 0x04000000 );
 }
+helpers.getPort = function() {
```

Comment:
Oh right, forgot this is in phantom. We should really consider trying nightmare...